docs/sphinx/source/introtutorial.rst

```diff
@@ -48,9 +47,29 @@ configuration at a handful of sites listed below.
     inverter = sapm_inverters['ABB__MICRO_0_25_I_OUTD_US_208__208V_']
     temperature_model_parameters = pvlib.temperature.TEMPERATURE_MODEL_PARAMETERS['sapm']['open_rack_glass_glass']
 
-    # specify constant ambient air temp and wind for simplicity
-    temp_air = 20
-    wind_speed = 0
+
+Let's download the typical meteorological year weather data from PVGIS, which
+includes irradiation, temperature and wind speed.  Note that PVGIS uses
+different naming conventions, so it is required to rename the weather data
+variables before using them. PVGIS weather data is already UTC-localized, so
+conversion to local timezone is optional.
```

Comment:
The text itself is pretty neutral, but I don't want to imply that we favor PVGIS. I'm wondering if we need a sentence here like ``See :ref:`iotools` for other weather data sources.`` or maybe even something more explicit.

Reply:
Pushed a [fixup commit with a little change](https://github.com/pvlib/pvlib-python/pull/1144/commits/c86a71c5385db21487dd3d2e6a175df3815a6298). Feel free to rephrase it as you'd like if you still prefer some other way to say it. :blush: